src_py/apiServer/apiServer.py

```diff
@@ -13,17 +13,22 @@ def __init__(self):
         self.mainServerAddress = 'http://' + mainServerIP + ':' + mainServerPort
 
         # Send the content of jsonPath to each devices:
-        print("Sending jsonPath to devices...\n")
+        print("Sending JSON paths to devices...")
+
+        archiAddress = globe.content[0]
+        connMapAddress = globe.content[1]
+        data = archiAddress + '#' + connMapAddress
 
         for ip in globe.components.devicesIp:
-            response = requests.post(ip + "/8484/updateJsonPath", data=globe.content)
+            address = 'http://{}:8484/updateJsonPath'.format(ip)
```

Comment:
@dordor7  There is a better formating approach in python 
f'http://{ip}:8484/updateJsonPath'
Moreover why 8484 port number is not a parameter?